src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,170 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private final String errorMassage = "Index: %d , Size: %s";
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
class fields have default values